src/libs/estimate/errors/estimateErrors.test.ts

```diff
@@ -0,0 +1,84 @@
+// eslint-disable-next-line max-classes-per-file
+import { ethers } from 'hardhat'
+
+import { describe, expect } from '@jest/globals'
+
+import { humanizeEstimationError } from './index'
+
+const MockBundlerAndPaymasterError = class extends Error {
+  public constructor(public shortMessage?: string) {
+    super(`UserOperation reverted during simulation with reason: ${shortMessage}`)
+  }
+}
+
+describe('Estimation errors are humanized', () => {
+  let contract: any
+
+  beforeEach(async () => {
+    const contractFactory = await ethers.getContractFactory('MockContract')
+    contract = await contractFactory.deploy()
+  })
+
+  it('Expired swap (Uniswap)', async () => {
+    try {
+      await contract.revertWithReason('Transaction too old')
+    } catch (error: any) {
+      const humanizedError = humanizeEstimationError(error)
+
+      expect(humanizedError.message).toBe(
+        'The transaction cannot be sent because the swap has expired. Return to the dApp and reinitiate the swap if you wish to proceed.'
+      )
+    }
+  })
+  it('Rpc timeout', () => {
+    const error = new Error('rpc-timeout')
+    const humanizedError = humanizeEstimationError(error)
+
+    expect(humanizedError.message).toBe(
+      'There seems to be a problem with the RPC on this network. Please try again later, change the RPC or contact support for assistance.'
+    )
+  })
+  it('Relayer error', () => {
+    const errorMessages = ['no json in res', 'Relayer error', 'failed to fetch']
+
+    errorMessages.forEach((errorMessage) => {
+      const error = new Error(errorMessage)
+      const humanizedError = humanizeEstimationError(error)
+
+      expect(humanizedError.message).toBe(
+        'Transaction cannot be sent because of an unknown error. Please try again or contact Ambire support for assistance.'
```

Comment:
Here, we know it's a relayer error but we don't say it to the user. Let's go and do something like:
* The Ambire relayer is currently down OR
OR
* The Ambire server is currently experiencing technical difficulties

Of course, we keep "Please try again or contact Ambire support for assistance"